oneanddone/functional_tests/requirements.txt

```diff
@@ -0,0 +1,4 @@
+bidpom
+pytest-selenium
+requests==2.8.1
+-e git+https://github.com/bobsilverberg/pytest-django.git@6d8d45c4fd1eb58b553045dd844e35885fcf67e6#egg=pytest_django
```

Comment:
I'm surprised if One and Done is doing anything radically different form other apps using this plugin, but I don't know anything about the plugin or even Django itself. I'll follow the issue you've raised.